src/main/java/org/fungover/storm/server/Server.java

```diff
@@ -40,10 +42,13 @@ public void acceptConnections(ServerSocket serverSocket) {
     }
 
     public static void main(String[] args) {
+        ConfigurationManager.getInstance().readConfigurationFile("src/main/resources/config.json");
```

Comment:
Have you tried this when running the application inside docker? The project now has a Dockerfile so it's possible to build a docker image and run it as a container.

Reply:
No have not tried,  but will look into it